spiceaidocs/docs/features/configurable-llms/runtime_tools.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Giving Language Models Runtime Tools'
+sidebar_label: 'Runtime tools'
+description: 'Learn how LLMs can interact with the spice runtime.'
+sidebar_position: 2
+pagination_prev: null
+pagination_next: null
+---
+
+Spice provides a fully OpenAI compatible [`v1/chat/completion`](/api/http/chat-completions) endpoint.
```

Comment:
**question:** Does this line need to be here? This doc file is about LLMs interacting with the Runtime, not SDKs interacting with the Runtime to invoke LLMs.